common/display/displayqueue.cpp

```diff
@@ -349,7 +349,9 @@ void DisplayQueue::GetCachedLayers(const std::vector<OverlayLayer>& layers,
                    !target_plane.IsVideoPlane()) {
           const OverlayLayer& layer = layers.at(source_layers.at(0));
           if (layer.IsVideoLayer()) {
+#ifndef DISABLE_VA
             target_plane.SetVideoPlane(true);
```

Comment:
Instead of commenting out all SetVideoPlane calls add this check in SetVideoPlane itself.

Reply:
reasonable. Will do